met-web/src/components/engagement/form/EngagementWidgets/AddContactDrawer.tsx

```diff
@@ -17,40 +17,74 @@ import { WidgetDrawerContext } from './WidgetDrawerContext';
 const schema = yup
     .object({
         name: yup.string().required(),
-        title: yup.string().required(),
-        phone_number: yup.string().required(),
+        title: yup.string(),
+        phone_number: yup.string(),
         email: yup.string().email().required(),
-        address: yup.string().required(),
-        bio: yup.string().min(20).required(),
+        address: yup.string(),
+        bio: yup.string(),
     })
     .required();
 
 type ContactForm = yup.TypeOf<typeof schema>;
 
 const AddContactDrawer = () => {
     const dispatch = useAppDispatch();
-
+    const { addContactDrawerOpen, handleAddContactDrawerOpen, loadContacts, selectedContact } =
+        useContext(WidgetDrawerContext);
+    const [isCreatingContact, setIsCreatingContact] = useState(false);
     const methods = useForm<ContactForm>({
         resolver: yupResolver(schema),
+        defaultValues: {
+            name: selectedContact ? selectedContact.name : '',
+            phone_number: selectedContact ? selectedContact.phone_number : '',
+            title: selectedContact ? selectedContact.title : '',
+            email: selectedContact ? selectedContact.email : '',
+            address: selectedContact ? selectedContact.address : '',
+            bio: selectedContact ? selectedContact.bio : '',
+        },
     });
 
+    useEffect(() => {
+        methods.setValue('name', selectedContact?.name);
+        methods.setValue('phone_number', selectedContact?.phone_number);
+        methods.setValue('title', selectedContact?.title);
+        methods.setValue('email', selectedContact?.email);
+        methods.setValue('address', selectedContact?.address);
+        methods.setValue('bio', selectedContact?.bio);
+    }, [selectedContact]);
```

Comment:
Is the defaultValues: { ... alone not working?

Reply:
No it is not, not sure why

Reply:
Okay well it's kinda weird that we need to do this. We need to look into it